packages/pages/src/build/build.ts

```diff
@@ -1,17 +1,15 @@
-import shell from "shelljs";
 import { CommandModule } from "yargs";
 import { ProjectFilepaths } from "../common/src/project/structure.js";
+import { build } from "vite";
 
 type BuildArgs = Pick<ProjectFilepaths, "scope">;
 
 const handler = async ({ scope }: BuildArgs) => {
   // Pass CLI arguments as env variables to use in vite-plugin
-  shell.exec("vite build", {
-    env: {
-      ...process.env,
-      YEXT_PAGES_SCOPE: scope,
-    },
-  });
+  if (scope) {
+    process.env.YEXT_PAGES_SCOPE = scope;
+  }
+  build();
```

Comment:
When the `env` argument is passed to `shell.exec`, does it make the environment variables available to the child process under `process.env`? I'm only asking because I want to be sure it's okay that we're using `process.env` here and not a de-structured version.

Reply:
Yes, `plugin.ts` accesses the env var as `process.env.YEXT_PAGES_SCOPE` even before this change, which is why `process.env` needed to be modified directly. Adding a variable to `process.env` like this only affects the current process, so after running the build script, `process.env.YEXT_PAGES_SCOPE` returns to being undefined if it wasn't defined before